### PR TITLE
Issue #39, Added requested denominations for magic scrolls in DFRPG i…

### PR DESCRIPTION
**Pull request by pgaurons**
…tem library. issue #39

### CODE REVIEW

File: Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment.eqp [hunk 1]
```diff
@@ -11725,6 +11917,12 @@
 		<prereq_list all="yes">
 			<contained_weight_prereq has="yes" compare="at most">8 lb</contained_weight_prereq>
 		</prereq_list>
+		<equipment version="6">
```
Comment (richardwilkes) [inline Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment.eqp:11920]: Why was this added?

**Comment by pgaurons** [comment]
I just toggled a scroll belt open because I was checking to see if there
was any documentation indicating how heavy a scroll should be.

**Comment by richardwilkes** [comment]
I was referring to the addition of an item named "Equipment" inside the container. I don't see the point of it and assume it was a mistake.

**Comment by pgaurons** [comment]
oh, if that is the case, let me fix it.

**Comment by pgaurons** [comment]
Should be fixed now.
